Common subdirectories: uffi-1.3.6.orig/benchmarks and uffi-1.3.6/benchmarks
Common subdirectories: uffi-1.3.6.orig/doc and uffi-1.3.6/doc
Common subdirectories: uffi-1.3.6.orig/examples and uffi-1.3.6/examples
Common subdirectories: uffi-1.3.6.orig/src and uffi-1.3.6/src
Common subdirectories: uffi-1.3.6.orig/tests and uffi-1.3.6/tests
diff -u uffi-1.3.6.orig/uffi-tests.asd uffi-1.3.6/uffi-tests.asd
--- uffi-1.3.6.orig/uffi-tests.asd	2003-08-14 16:40:54.000000000 -0500
+++ uffi-1.3.6/uffi-tests.asd	2003-10-15 17:44:44.783076368 -0500
@@ -14,6 +14,7 @@
   (:use #:asdf #:cl))
 (in-package #:uffi-tests-system)
 
+#+(or allegro lispworks cmu mcl cormanlisp sbcl scl)
 (defsystem uffi-tests
     :depends-on (:uffi)
     :components
@@ -37,6 +38,7 @@
 	       (:file "foreign-var" :depends-on ("foreign-loader"))
 	       ))))
 
+#+(or allegro lispworks cmu mcl cormanlisp sbcl scl)
 (defmethod perform ((o test-op) (c (eql (find-system :uffi-tests))))
   (or (funcall (intern (symbol-name '#:do-tests)
 		       (find-package '#:regression-test)))
Only in uffi-1.3.6: uffi-tests.asd~
